src/main/resources/org/sonatype/nexus/ci/iq/ManualApplication/config.groovy

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.iq.ManualApplication
+
+import org.sonatype.nexus.ci.iq.Messages
+
+def f = namespace(lib.FormTagLib)
+
+f.entry(title: _(Messages.IqPolicyEvaluation_Application()),field: 'applicationIdEntry') {
```

Comment:
`help: descriptor.getHelpFile(` rather than `field: ` for a fake field

Reply:
>help: descriptor.getHelpFile( rather than field: for a fake field

Changed 4abfac4